gprofiler/profilers/__init__.py

```diff
@@ -1,8 +1,13 @@
 # NOTE: Make sure to import any new process profilers to load it
-from gprofiler.profilers.java import JavaProfiler
-from gprofiler.profilers.perf import SystemProfiler
-from gprofiler.profilers.php import PHPSpyProfiler
 from gprofiler.profilers.python import PythonProfiler
-from gprofiler.profilers.ruby import RbSpyProfiler
+from granulate_utils import is_windows
+if not is_windows():
+    from gprofiler.profilers.java import JavaProfiler
+    from gprofiler.profilers.perf import SystemProfiler
+    from gprofiler.profilers.php import PHPSpyProfiler
+    from gprofiler.profilers.ruby import RbSpyProfiler
+from gprofiler.profilers.python_ebpf import PythonEbpfProfiler
 
-__all__ = ["JavaProfiler", "SystemProfiler", "PHPSpyProfiler", "PythonProfiler", "RbSpyProfiler"]
+__all__ = ["JavaProfiler", "SystemProfiler", "PHPSpyProfiler", "PythonProfiler", "RbSpyProfiler"] if not is_windows() else ["SystemProfiler", "PythonProfiler"]
```

Comment:
I'd structure it as `__all__ = '["SystemProfiler", "PythonProfiler"]` and under the `is_windows()` branch, `__all__ += [...]`